src/index.js

```diff
@@ -31,76 +26,13 @@ export default class Kinto {
     };
   }
 
-  /**
-   * Synchronization strategies. Available strategies are:
-   *
-   * - `MANUAL`: Conflicts will be reported in a dedicated array.
-   * - `SERVER_WINS`: Conflicts are resolved using remote data.
-   * - `CLIENT_WINS`: Conflicts are resolved using local data.
-   *
-   * @type {Object}
-   */
-  static get syncStrategy() {
-    return Collection.strategy;
-  }
-
-  /**
-   * Constructor.
-   *
-   * Options:
-   * - `{String}`       `remote`      The server URL to use.
-   * - `{String}`       `bucket`      The collection bucket name.
-   * - `{EventEmitter}` `events`      Events handler.
-   * - `{BaseAdapter}`  `adapter`     The base DB adapter class.
-   * - `{String}`       `dbPrefix`    The DB name prefix.
-   * - `{Object}`       `headers`     The HTTP headers to use.
-   * - `{String}`       `requestMode` The HTTP CORS mode to use.
-   *
-   * @param  {Object} options The options object.
-   */
   constructor(options={}) {
     const defaults = {
       adapter: Kinto.adapters.IDB,
-      bucket: DEFAULT_BUCKET_NAME,
-      events: new EventEmitter(),
-      remote: DEFAULT_REMOTE,
+      events: new EventEmitter()
```

Comment:
Maybe we could allow users to pass in their own `events` instance? The previous jsdoc was misleading btw.